roles/kubernetes/control-plane/vars/main.yaml

```diff
@@ -7,3 +7,11 @@ kube_apiserver_admission_plugins_needs_configuration:
 - PodSecurity
 - PodNodeSelector
 - ResourceQuota
+
+kubeadm_init_first_control_plane_cmd: >-
```

Comment:
Sorry, should have been more clear, I meant a var at the block level, since it's only used there.

Reply:
Understood.

Reply:
Moved to block vars.